### PR TITLE
debug/log: add log level for JSON type

### PR DESCRIPTION
**Pull request by e-urban**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
    - Note that I did not make a request to get the developer role but still was able to assign the issue to myself.
- [x] I have signed the Open Information Security Foundation contribution agreement at https://suricata-ids.org/about/contribution-agreement/
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)
    - N/A I could not find any documentation related to this change.

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/2671

Describe changes:
- Add log level to the JSON type Suricata log

[PRScript](https://redmine.openinfosecfoundation.org/projects/suricata/wiki/PRscript) output (if applicable):

### CODE REVIEW

**Comment by e-urban** [comment]
Ug, I just realized I put the log_level after the event_type in my pull request, which is not the correct ordering.  This commit should correctly start on line 225 so that it is before the event_type.  

I believe to correct my mistake that I should create a loglevel-json-2671-v2 branch.  Rather than do that now and make things more messy than they should be, I will await any further feedback from the review of this pull request and will incorporate changes there.

File: src/util-debug.c [hunk 1]
```diff
@@ -224,6 +224,14 @@ static int SCLogMessageJSON(struct timeval *tval, char *buffer, size_t buffer_si
 
     json_object_set_new(js, "event_type", json_string("engine"));
 
+    const char *s = NULL;
+    s = SCMapEnumValueToName(log_level, sc_log_level_map);
```
Comment (victorjulien) [inline src/util-debug.c]: the variable declaration could be on this line as well, makes the code a bit more compact

**Comment by victorjulien** [comment]
Hi Eric, just a minor comment. Other than that I noticed that your author name is "Author: e-urban" instead of your real name. Is that intentional?

File: src/util-debug.c (hunk 1 above)
Comment (e-urban) [reply src/util-debug.c]: Corrected in #3558.

**Comment by e-urban** [comment]
Hello Victor, that was not intentional to use e-urban as the author, so thank you for letting me know.  I updated my git config before doing the commit on the pull request that replaces this.

**Comment by e-urban** [comment]
This is replaced by #3558.

**Pull request closed without merging**